app/controllers/admin/services_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Admin
+
+  class ServicesController < ApplicationController
+
+    def cancel_reservations_for_offline_instruments
+      InstrumentOfflineReservationCanceler.new.cancel!
+      render nothing: true
```

Comment:
Right, my assumption is the client is a cron job running `curl` so a standard 500 seemed reasonable for now.